t/test/mojo.t

```diff
@@ -0,0 +1,118 @@
+use Mojo::Base -strict;
```

Comment:
I'm not sure if this file is required. What does it really test? I can't see that we test Test::Mojo directly anywhere.